src/ProfilesList.json

```
@@ -143,5 +143,6 @@
   "nischalon10.json",
   "Axorax.json",
   "Mannuel25.json",
+  "Vedant91827",
```

```suggestion
  "Vedant91827.json",
```
Add complete file name.